pennylane/gradients/parameter_shift_cv.py

```diff
@@ -450,7 +450,8 @@ def processing_fn(results):
         start = 0
 
         if not results:
-            results = [np.squeeze(np.zeros([tape.output_dim]))]
+            _output_dim = 1
+            results = [np.squeeze(np.zeros([_output_dim]))]

```

Comment:
```suggestion
            results = np.array(0.)

```
Seems like extra complication for no reason if its just always a 1